app/Jobs/Server/UpdateKernelJob.php

```
@@ -0,0 +1,59 @@
+
+    public function failed(Exception $e): void
+    {
+        Notifier::send($this->server, new ServerUpdateFailed($this->server));
+        $this->server->checkConnection();
+        $this->broadcastServerUpdate();
+
+        ServerLog::log(
+            $this->server,
+            'update-server-failed',
+            $e->getMessage()
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Models\ServerLog' in method 'failed'. (undefined)

(StaticAccess)